integration-tests/features/steps/stack_analysis.py

```diff
@@ -110,9 +110,12 @@ def stack_analysis_endpoint(context, version):
     """Return endpoint for the stack analysis of selected version."""
     # Two available endpoints for stack analysis are /stack-analyses and /analyse
     # /analyse endpoint was developed to meet the performance norms at production
-    endpoint = {"1": "/api/v1/stack-analyses-v1",
-                "2": "/api/v1/analyse",
-                "3": "/api/v1/stack-analyses"}.get(version)
+    endpoint_arr = ["/api/v1/stack-analyses-v1",
+                    "/api/v1/analyse",
+                    "/api/v1/stack-analyses/"]
+    index = int(version) - 1
+    endpoint = endpoint_arr[index]
```

Comment:
Please note that this statement can fail now. Before it was checked by next statement `if endpoint is None:`

Reply:
also, we can force Behave to store version as an int.